araproc/analysis/cw_filter.py

```diff
@@ -64,6 +64,10 @@ def apply_filters(cw_filters, waveform_bundle, cw_ids = None, min_cw_id_freq = 0
         A tuple of numpy arrays containing cw id info.
         If None, all set CW filters allowed to filter. Otherwise, only those 
         covering frequencies in the cw id info will be allowed to filter.
+    min_cw_id_freq : float
```

Comment:
Can we name this variable better please? Maybe "always_on_min_cw_id_freq? Ditto for the max below? I know it's in the docs, but I think it's otherwise confusing.